source/release-notes.txt

```diff
@@ -10,6 +10,34 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.40.3
+----------------
+
+*Released October 11, 2023*
+
+New Features:
+
+- After creating a $search index you are now redirected to the search 
+  indexes modal (:issue:`COMPASS-7247`).
+- Field names are now autocompleted when defining indexes. (:issue:`COMPASS-7174`).
+- Syntax errors are now highlighted when defining indexes. (:issue:`COMPASS-7246`).
+- Improved AI feedback experience (:issue:`COMPASS-7211`, :issue:`COMPASS-7251`).
+- Removed ``useNewUrlParser`` and ``useUnifiedTopology`` from export 
+  to language options (:issue:`COMPASS-4897`).
+- The knnVector index template definition is now used instead of 
+  vectorEmbedding (:issue:`COMPASS-7288`).
+
+Bug Fixes:
+
+- Fixed GUI flicker when closing search index modal (:issue:`COMPASS-7248`).
+- Downgraded electron to version 25.8.4 (:issue:`COMPASS-7291`).
```

Comment:
(nit / suggestion)
```suggestion
- Downgraded Electron to version 25.8.4 (:issue:`COMPASS-7291`).
```

suggestion:
What about linking out to Electron: https://www.electronjs.org/ ? I had to google to discover what it was. IDK if I'm common in this regard, but it might be useful. CC: @jaoppenheim for feedback